### DISCUSSION TITLE
K9-Mail possiple issue with tags

### DISCUSSION THREAD

**Discussion by gaabster** [ImranR98/Obtainium#1641] [Q&A] [answered]
Hello.
Obtainium installs the **pre-release** version of k9-mail despite the **Verify the 'latest' tag** option enabled. I suspect this is an issue with GitHub tags but I could be mistaken. Is there something I can do here?
Thank You.

**Comment by ImranR98** [comment]
What version do you get? It installed 6.803 on my end with default settings.

**Reply by gaabster** [reply, marked as answer]
Oh, I have totally forgot about this post here. I found the issue, it was play store updating the app because I was in beta program.
I apologize for not reporting it sooner and wasting your precious time.
Kind regards,
Gabriel.
